### PR TITLE
add updating and retrieving connector integrations

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

File: src/main/java/com/gooddata/connector/Integration.java [hunk 1]
```diff
@@ -30,7 +30,11 @@
     private final IntegrationProcessStatus runningProcess;
 
     public Integration(final String projectTemplate) {
-        this(projectTemplate, true, null, null, null);
+        this(projectTemplate, true);
+    }
+
+    public Integration(final String projectTemplate, final boolean active) {
+        this(projectTemplate, active, null, null, null);
```
Comment (martiner) [inline src/main/java/com/gooddata/connector/Integration.java]: actually the missing constructor is intentional, i don't think we need a constructor for everything as we have a setter...

File: src/main/java/com/gooddata/connector/ConnectorService.java [hunk 2]
```diff
@@ -34,6 +34,25 @@ public ConnectorService(final RestTemplate restTemplate, final ProjectService pr
...
+     * @return              integration
+     * @throws ConnectorException if integration can't be retrieved
+     */
+    public Integration getIntegration(final Project project, final ConnectorType connectorType) {
+        notNull(project, "project");
+        notNull(connectorType, "connector");
+
+        try {
+            return restTemplate.getForObject(Integration.URL, Integration.class, project.getId(), connectorType.getName());
+        } catch (GoodDataRestException | RestClientException e) {
```
Comment (martiner) [inline src/main/java/com/gooddata/connector/ConnectorService.java]: what if integration wasn't yet created? `IntegrationNotFoundException`?

File: src/main/java/com/gooddata/connector/Integration.java (hunk 1 above)
Comment (sadam21) [inline src/main/java/com/gooddata/connector/Integration.java]: OK, removed...

File: src/main/java/com/gooddata/connector/ConnectorService.java (hunk 2 above)
Comment (sadam21) [inline src/main/java/com/gooddata/connector/ConnectorService.java]: Added...

**Comment by martiner** [comment]
close #108

**Pull request merged**